sg/path.go

```diff
@@ -15,6 +15,10 @@ const (
 	SageFileBinary = "sagefile"
 )
 
+const (
+	buildDir = "build"
```

Comment:
Is this in its own group because it’s private?

Reply:
yes, but now I pushed an update where we make them all private!

Reply:
Sweet!